src/components/WorkPoolQueueDetails.vue

```diff
@@ -6,6 +6,16 @@
       </template>
     </p-key-value>
 
+    <p-key-value label="Status" :alternate="alternate">
+      <template #value>
+        <WorkQueueStatusBadge v-if="workPool" :work-queue="workPoolQueue" :work-pool="workPool" />
+      </template>
+    </p-key-value>
+
+    <template v-if="workQueueStatus">
+      <p-key-value label="Last Polled" :value="workQueueStatus.lastPolled ? formatDateTimeNumeric(workQueueStatus.lastPolled) : null" :alternate="alternate" />
```

Comment:
Can you move this value logic out of the template? It'll be easier to understand there